src/pages/doi.org/[...doi].tsx

```diff
@@ -477,10 +484,186 @@ const WorkPage: React.FunctionComponent<Props> = ({ doi, metadata }) => {
   const work = data.work
 
   const content = () => {
+    const exportMetadata = () => {
```

Comment:
@jrhoads I think this probably should be its own component. Don't you think? IT's a lot of code

Reply:
I agree.